tests/drum/test_units.py

```diff
@@ -700,3 +700,26 @@ def test_run_java_server_entry_point_succeed(self):
         # required to properly shutdown py4j Gateway
         pred._setup_py4j_client_connection()
         pred._stop_py4j()
+
+
+@pytest.mark.parametrize("data_dtype,label_dtype", [(int, int), (float, int)])
```

Comment:
how about `(int, float)`?

Reply:
so we're going to just not allow support float labels.